RatScanner/View/MainWindow.xaml

```diff
@@ -11,6 +11,18 @@
         ResizeMode="CanMinimize"
         Height="400" Width="300">
 
+  <Window.Resources>
+    <Style TargetType="{x:Type Hyperlink}">
+      <Setter Property="TextBlock.FontSize" Value="14" />
+      <Setter Property="Foreground" Value="White" />
```

Comment:
Missing `<Setter Property="FontFamily" Value="Roboto Medium"/>` for styling like other text or is this a deliberate choice?

Reply:
Added missing props